components/tikv_util/src/config.rs

```diff
@@ -221,9 +221,9 @@ impl<'de> Deserialize<'de> for ReadableSize {
 #[derive(Clone, Debug, PartialEq)]
 pub struct ReadableDuration(pub Duration);
 
-impl Into<Duration> for ReadableDuration {
-    fn into(self) -> Duration {
-        self.0
+impl From<ReadableDuration> for Duration {
```

Comment:
I'm curious why this works. LOL
Doesn't it violate the orphan rule? And the official documents also mentioned we can only implement `Into` if we want to convert to an external type.